### DISCUSSION TITLE
Cannont paste a full web page into a new document

### DISCUSSION THREAD

**Discussion by Rex-Lin-TW** [outline/outline#4230] [General]
I try to copy a webpage and paste to outline's document, I found that it has a limitation. If the web pate has just few content, there is no problem. But if the web page has a little more content, we face a problem.

We cannot copy whole web page and paste to the outline document. Otherwise, only the text paste to outline document but the images, format are gone.  The workaround is copy partial content and paste to the outline and do this several times for a single web page, it is very inconvenience.

For example, the web page https://www.itpro.co.uk/network-internet/wifi-hotspots/362157/ubiquiti-networks-unifi-u6-lr-review-a-wealth-of-features-for
We cannot copy whole page and paste it to outline in one time. But if we separate it as three parts, do copy & paste three times, then it can be done.

Is it caused by an incorrect setup?  Or it is a nature behavior of outline?

Thank you.

**Comment by tommoor** [comment]
Perhaps you could provide a video of the behavior. Outline only supports a small subset of what's possible in webpages, it's not really designed to have an entire website pasted in for sure so It's not surprising if there is a bug or two here :)

**Reply by Rex-Lin-TW** [reply]
Hi Sir,

Sorry, I am not familiar about the video sharing, it took me some times. Finally, I make a sample video as below link.
As you can see in the video, if I copied full page, only the plaint text was pasted, but if I copied a small piece content with text and image, it can be fully pasted to the outline.
[https://sendvid.com/7exmsxth](url)

Thank you.

PS. The video link will be kept 90 days only because of the limitation of this free service.

**Reply by tommoor** [reply]
Gotcha, it seems that some content further down this page is causing it to be false-detected as Markdown which means it skips the html content. Will fix

**Reply by Rex-Lin-TW** [reply]
Hi Sir,

Some more feedback.

The URL of my sample web page is below.
If I copy it partially, no matter which portion of this web page I copied, it can be pasted with the HTML format, 
[https://evanmccann.net/blog/2021/2/unifi-6-lite-review](url)

Just for your reference, thank you.

**Reply by tommoor** [reply]
This issue has been fixed here:
https://github.com/outline/outline/commit/a8936039e5ec1278ea0a8397f4af75b295330ed1

It is live on getoutline.com and will be in the next community release.
